src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -502,6 +502,79 @@ g.test('default_blend_constant,initial_blend_constant')
     t.expectOK(result);
   });
 
+g.test('default_blend_constant,setting_blend_constant')
+  .desc(`Test that setting the blend constant to the RGBA values works at the beginning of a pass.`)
+  .paramsSubcasesOnly([
+    { r: 1.0, g: 1.0, b: 1.0, a: 1.0 },
+    { r: 0.5, g: 1.0, b: 0.5, a: 0.0 },
+    { r: 0.0, g: 0.0, b: 0.0, a: 0.0 },
+  ])
+  .fn(async t => {
+    const { r, g, b, a } = t.params;
+
+    const format = 'rgba8unorm';
+    const kSize = 1;
+    const kWhiteColorData = new Float32Array([255, 255, 255, 255]);
+    const kBlackColorData = new Float32Array([0, 0, 0, 0]);
+
+    const basePipeline = t.createRenderPipelineForTest({ format }, undefined);
+    const testPipeline = t.createRenderPipelineForTest(
+      { format },
+      {
+        srcFactor: 'constant',
+        dstFactor: 'one',
+        operation: 'add',
+      }
+    );
```

Comment:
Will need rebase over #1986 or vice versa

Reply:
Thank you for updating the previous PR. Let me rebase this PR over #1986.